### PR TITLE
Bump plugin parent POM

### PR DESCRIPTION
**Pull request by basil**
Bumps the [plugin parent POM](https://github.com/jenkinsci/plugin-pom) from 3.48 to 3.54.

See [the changelog](https://github.com/jenkinsci/plugin-pom/releases) and the full diff in [the compare view](https://github.com/jenkinsci/plugin-pom/compare/plugin-3.48...plugin-3.54).

### CODE REVIEW

**Comment by basil** [comment]
`TeeStepTest.smokes` fails on Windows with the following stack trace:

```
java.lang.AssertionError: 

Expected: a string containing "got: first message second message WS>rem"
     but: was "Started
Running in Durability level: MAX_SURVIVABILITY
[Pipeline] node
Running on remote in C:\Jenkins\workspace\eline-utility-steps-plugin_PR-72\target\tmp\j h8512543174852602852\workspace\p
[Pipeline] {
[Pipeline] dir
Running in C:\Jenkins\workspace\eline-utility-steps-plugin_PR-72\target\tmp\j h14870851017203151330\workspace\p
[Pipeline] {
[Pipeline] tee
[Pipeline] {
[Pipeline] echo
first message
[Pipeline] semaphore
Resuming build at Wed Dec 18 22:23:33 UTC 2019 after Jenkins restart
Waiting to resume part of p #1: Finished waiting
Waiting to resume part of p #1: ‘Jenkins’ doesn’t have label ‘remote’
‘remote’ is offline
Ready to run at Wed Dec 18 22:23:36 UTC 2019
[Pipeline] echo
second message
[Pipeline] isUnix
[Pipeline] bat
[Pipeline] }
[Pipeline] // tee
[Pipeline] readFile
[Pipeline] pwd
[Pipeline] echo
got: first message second message jenkins@win2019-1d7e20 WS>rem
[Pipeline] }
[Pipeline] // dir
[Pipeline] }
[Pipeline] // node
[Pipeline] End of Pipeline
Finished: SUCCESS
"
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
	at org.junit.Assert.assertThat(Assert.java:956)
	at org.junit.Assert.assertThat(Assert.java:923)
	at org.jvnet.hudson.test.JenkinsRule.assertLogContains(JenkinsRule.java:1402)
	at org.jenkinsci.plugins.pipeline.utility.steps.fs.TeeStepTest.lambda$smokes$1(TeeStepTest.java:75)
	at org.jvnet.hudson.test.RestartableJenkinsRule$3.evaluate(RestartableJenkinsRule.java:232)
	at org.jvnet.hudson.test.RestartableJenkinsRule$6.evaluate(RestartableJenkinsRule.java:272)
	at org.jvnet.hudson.test.JenkinsRule$1.evaluate(JenkinsRule.java:598)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

That failure doesn't seem related to this change, since it also occurred in #71. I suspect the same would occur on `master` as well, if a build were run again today.

As far as I can tell, this change will be ready to merge once `master` itself is clean.

**Comment by basil** [comment]
I recommend reviewing this change with the "Hide whitespace changes" option enabled.

File: pom.xml [hunk 1]
```diff
@@ -28,14 +28,14 @@
...
-        <version>3.48</version>
+        <version>3.54</version>
         <relativePath />
     </parent>
     <artifactId>pipeline-utility-steps</artifactId>
     <packaging>hpi</packaging>
     <version>${revision}${changelist}</version>
     <name>Pipeline Utility Steps</name>
-    <url>https://wiki.jenkins.io/display/JENKINS/Pipeline+Utility+Steps+Plugin</url>
+    <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
```
Comment (rsandell) [inline pom.xml]: To do this change you must first move the docs from the wiki to the readme. But I also don't like putting user docs in the root readme, that file is more imho for visitors of the repo itself. So the docs should be put in for example `docs/README.md`

Comment (basil) [reply pom.xml]: > To do this change you must first move the docs from the wiki to the readme.

I have no interest in doing this, so I will revert the change.

**Pull request merged**